### PR TITLE
Enable automated tests

### PR DESCRIPTION
**Pull request by vlebedeff**
## Summary

* Add `make test` target that runs the whole suite of specs in Docker
* Release a new patch version
* Prerequisite for https://github.com/Mixbook/mixbook_com/pull/11550. Will allow to add changes here that will make the library forward-compatible with Rails 4.2

## Next Steps

https://github.com/Mixbook/tagtical/pull/15

### CODE REVIEW

**Review by noku — APPROVED** [review]
nice job

File: Rakefile [hunk 1]
```diff
@@ -5,6 +5,7 @@ desc 'Default: run specs'
 task :default => :spec
 RSpec::Core::RakeTask.new do |t|
   t.pattern = "spec/**/*_spec.rb"
+  # t.rspec_opts = "--format documentation"
```
Comment (yuriihohan) [inline Rakefile:8]: Is a temporary change?

File: gemfiles/Gemfile-rails.3.2.x [hunk 2]
```diff
@@ -0,0 +1,9 @@
+source "https://rubygems.org"
+
+gemspec
+
+gem "rails", "~> 3.2.0"
```
Comment (yuriihohan) [inline gemfiles/Gemfile-rails.3.2.x:5]: Could you explain the value of keeping this file?

File: Rakefile (hunk 1 above)
Comment (vlebedeff) [reply Rakefile:8]: No. It's convenient to turn it off or on as you work on the lib. When it's on it makes it easy to see which example produced a warning.

File: gemfiles/Gemfile-rails.3.2.x (hunk 2 above)
Comment (vlebedeff) [reply gemfiles/Gemfile-rails.3.2.x:5]: No value really. I just copied the original `Gemfile` here. Can be removed

**Pull request merged**